### PR TITLE
Document the new ID column.

### PR DESCRIPTION
**Pull request by sherfert**
To be merged after https://github.com/neo-technology/neo4j/pull/16740

### CODE REVIEW

**Review by LinneaAndersson — APPROVED** [review]
Looks good 👍

**Comment by sherfert** [comment]
@davidoliverSP2 This is ready for review, the other PR is merged.

File: cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc [hunk 1]
```diff
@@ -35,6 +35,12 @@ Such operators need to complete execution in its entirety before any rows are se
 These operators are called _eager_ operators, and are denoted as such in <<execution-plan-operators-summary>>.
 Eagerness can cause high memory usage and may therefore be the cause of query performance issues.
 
+[[id-column]]
+.IDs
+Each operator is assigned a unique ID, which is shown in the execution plan.
+The IDs help referring unambiguously to operators.
+There are no guarantees about the order of IDs, although they will usually start with 0 at the root and increase toward the leaves of the tree.
```
Comment (davidoliverSP2) [inline cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc]: This seems a little vague:
> There are no guarantees about the order of IDs, although they will usually start with 0 at the root and increase toward the leaves of the tree.

```suggestion
The IDs can be used to refer unambiguously to operators.
There are no guarantees about the order of IDs, although they will usually start with 0 at the root, and will increase towards the leaves of the tree.
```

**Review by davidoliverSP2 — COMMENTED** [review]
Thanks @sherfert 

One small editorial suggestion.

**Review by davidoliverSP2 — APPROVED** [review]
👍  thanks @sherfert

**Comment by davidoliverSP2** [comment]
@sherfert does this docs change need to be cherry picked back to `5.0`?
I see the original PR (https://github.com/neo-technology/neo4j/pull/16740) was merged into 5.0

**Comment by sherfert** [comment]
@davidoliverSP2 The original PR was only merged into `5.1`.
The branch name starts with `5.0`, because I started that work before the cutoff.

**Pull request merged**